epitator/annospan.py

```diff
@@ -140,3 +134,37 @@ def iterate_leaf_base_spans(self):
         for span in self.iterate_base_spans():
             if not isinstance(span, SpanGroup):
                 yield span
```

Comment:
I disagree about it being the natural way to iterate over a SpanGroup. When iterating over a list that contains lists or a set that contains sets, child lists and child sets are not iterated over, so I think it goes against the conventions established by other objects.

Reply:
I see that argument. I think that there is an argument to be made that there are specifics of this data structure which mean that its behavior should differ from those general container classes, but I could see an argument for leaving it as a method you call rather than the default `__iter__`.